src/components/Documents/MyDocuments.tsx

```diff
@@ -174,9 +189,21 @@ class MyDocuments extends Component<Props, State> {
     const documentId = documentData[rowIndex].id;
     const documentName = documentData[rowIndex].filename;
 
-    const fileType = FileType.IDENTIFICATION_PDF;
     const targetUser = this.props.username;
 
+    let fileType;
+    if (
+        userRole === Role.Worker ||
+        userRole === Role.Admin ||
+        userRole === Role.Director
+    ) {
```

Comment:
shouldn't workers be able to download identification pdfs as well?

Reply:
Yes, they are able to, but remember that the way that Louis coded it is that the worker "becomes" the client, with the client's username and role, so the checking here is based on that

Reply:
ok